custom/nic_compliance/forms.py

```diff
@@ -0,0 +1,39 @@
+from django import forms
+from django.utils.translation import ugettext as _
+from django.core.exceptions import ValidationError
+from django.contrib.auth import password_validation
+
+
+class EncodedPasswordChangeForm(object):
```

Comment:
The name change sounds good.

About the doc string, this can be used by any forms using password fields, for ex: this mixin is currently being used by classes inheriting from `PasswordChangeForm` as well.
You saying this specifically for `clean_new_password1` method?